NtupleProducer/plugins/HTauTauNtuplizer.cc

```diff
@@ -2338,6 +2378,16 @@ int HTauTauNtuplizer::FillJet(const edm::View<pat::Jet> *jets, const edm::Event&
     _bdiscr13.push_back(ijet->bDiscriminator("pfDeepFlavourJetTags:probuds"));
     _bdiscr14.push_back(ijet->bDiscriminator("pfDeepFlavourJetTags:probg"));
 
+    // ParticleNet
+    _bdiscr15.push_back(ijet->bDiscriminator("pfParticleNetAK4JetTags:probbb"));
+    _bdiscr16.push_back(ijet->bDiscriminator("pfParticleNetAK4JetTags:probpu"));
+    _bdiscr17.push_back(ijet->bDiscriminator("pfParticleNetAK4JetTags:probcc"));
+    _bdiscr18.push_back(ijet->bDiscriminator("pfParticleNetAK4JetTags:probundef"));
+    _bdiscr19.push_back(ijet->bDiscriminator("pfParticleNetAK4JetTags:probc"));
+    _bdiscr20.push_back(ijet->bDiscriminator("pfParticleNetAK4JetTags:probb"));
+    _bdiscr21.push_back(ijet->bDiscriminator("pfParticleNetAK4JetTags:probuds"));
+    _bdiscr22.push_back(ijet->bDiscriminator("pfParticleNetAK4JetTags:probg"));
+
```

Comment:
Just double checking: whay are these called `AK4` when they are used for AK8 jets?

Reply:
I learned in the talk by Raffa at our MIB meeting that ParticleNet can tag also resolved jets ;)

Reply:
Ok cool" Thanks Davide! My question was more about the "correctness" of the names of the taggers?

Reply:
I assume these are the taggers for AK4 jets, the one for AK8 have AK8 in the name...

Reply:
I'll let @riga confirm, but these names look correct (and are defined [here](https://github.com/cms-sw/cmssw/blob/CMSSW_10_6_X/RecoBTag/ONNXRuntime/python/pfParticleNetAK4_cff.py))